components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.ts

```diff
@@ -21,6 +21,10 @@ export class ServicesSidebarComponent implements OnInit {
   public selectedHealth: string;
   public services$: Observable<Service[]>;
   public serviceGroupName$: Observable<string>;
+  currentPage = 1;
+  pageSize = 25;
+  // TODO: Wire this up with real total
+  totalServices = 50;
```

Comment:
We will be able to hook this total to my PR or vice-versa https://github.com/chef/automate/pull/136